spec/rails_helper.rb

```diff
@@ -13,9 +13,10 @@
 require "fakeredis/rspec"
 require "pundit/matchers"
 require "pundit/rspec"
-require "webmock/rspec"
 require "sidekiq/testing"
+require "test_prof/factory_prof/nate_heckler"
```

Comment:
only a question: should we disable it in CI? I think it would add value to know how much time is spent in `FactoryBot` overall, especially because devs almost never run the entire test suite locally

Reply:
I don't expect the profiler to add substantial time to CI (and rspec is still faster than the e2e tests so adding some time there won't slow the build down when you're looking at elapsed time). 

I say leave it enabled in CI.